src/commands/infra/generate.ts

```diff
@@ -7,7 +7,7 @@ import simpleGit from "simple-git/promise";
 import { logger } from "../../logger";
 import { copyTfTemplate } from "./scaffold";
 
-const spkTemplatesPath = path.join(os.homedir(), ".spk/templates");
+export const spkTemplatesPath = path.join(os.homedir(), ".spk/templates");
```

Comment:
This would be best handled in a common / shared source file rather than living in one command and then referenced from another.